Automattic/wp-calypso
# Woo: Remove variation edits after success

This removes variation edits as they complete successfully.

To test: `npm test`
Additional optional testing: Create a new product with variations and observe the redux edit state in dev tools afterwards. The variations should be gone from creates.
